run-qemu/run.sh

```diff
@@ -49,6 +49,14 @@ x86_64)
 	kvm_accel="-cpu kvm64 -enable-kvm"
 	tcg_accel="-cpu qemu64 -machine accel=tcg"
 	;;
+aarch64)
+	qemu="qemu-system-aarch64"
+	console="ttyAMA0,115200"
+	smp=$(nproc)
+	kvm_accel="-cpu max -machine virt,accel=kvm:tcg"
+	tcg_accel="-cpu cortex-a72 -machine virt,accel=kvm:tcg"
```

Comment:
have you tried `-cpu host` instead of specifying exact CPU model? This would let qemu do faster emulation as it won't have to emulate different CPU instruction set?

Reply:
I remember trying `-cpu max` fix mixed success. I can give this another spin. I was under the impression that because kvm is not available, it would just have to do total emulation anyway?